src/js/media/html5.js

```diff
@@ -184,6 +193,39 @@ vjs.Html5.prototype.hideCaptions = function() {
   }
 };
 
+vjs.Html5.prototype.addInBandMetadataTracks = function(event) {
+  var tracks = this.textTracks(),
+      newTrack = event.track || null,
+      track,
+      i = 0;
+
+  for (i = 0; i < tracks.length; i++) {
```

Comment:
Yeah, I'm not certain it'll be necessarily useful. Even if you re-apply the same video you get a new track.
This was kind of left-over from when I had the removal of tracks in the same method as additions and so far I left it for "extra security" but I'm not certain it actually buys us much.